src/pages/guide/language/types.md

```diff
@@ -18,7 +18,7 @@ let myInt = 5;
 let myInt = (5 : int);
 let myInt = (5 : int) + (4 : int);
 let add (x: int) (y: int) :int => x + y;
-let drawCircle radius::(r: int) :unit => ...;
+let drawCircle radius::(r: int) :unit => ...; /* radius::(r: int) is a labeled argument. More on this [here](https://reasonml.github.io/guide/language/functions) */
```

Comment:
Ideally I wish we could have fragment identifiers for subheadings. Labeled arguments could be mentioned directly.